src/main/java/com/mentoring/ecommerce/adapter/in/web/ProductController.java

```diff
@@ -15,13 +18,24 @@
 @RequestMapping("/api/products")
 public class ProductController {
 
+    @Autowired
```

Comment:
I put @Autowired because was not working without them.
I'm still tracing this problem.